api/transport/handler.go

```diff
@@ -119,62 +115,50 @@ type StreamHandler interface {
 
 // DispatchUnaryHandler calls the handler h, recovering panics and timeout errors,
 // converting them to yarpc errors. All other errors are passed trough.
+// Deprecated: Use InvokeUnaryHandler instead.
```

Comment:
I believe the standard is to have an empty, commented out new line before deprecated notices:
```
// converting them to yarpc errors. All other errors are passed trough.
//
// Deprecated: Use InvokeUnaryHandler instead.
```

eg: https://github.com/yarpc/yarpc-go/blob/master/yarpcerrors/errors.go#L359